salt/example.top

```diff
@@ -9,10 +9,11 @@ base:
         - elife.gearman
         - elife.aws-credentials
         - elife.aws-cli
-        - elife.java8
+        - elife.openjdk8
         - search.elasticsearch
         - search.gearman-persistence
         - search
+        - search.leader
```

Comment:
needs to be added https://github.com/elifesciences/builder-configuration/blob/master/salt/top.sls#L367-L373 upon merge